### PR TITLE
Close search only

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
### QA
- Click on search
- See that the search and midpoint/whitebox (sectionC) opens
- Click the X close icon at the top right
- The search should reset and shrink
- Click on the search input and it should grow again
- Click anywhere outside the whitebox and it should close completely

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3850/
Test PASSed.

**Comment by kadams54** [comment]
QA is :ok: and code is :+1:.

**Comment by makyo** [comment]
:+1:!

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

File: jujugui/static/gui/src/app/components/header-search/test-header-search.js [hunk 1]
```diff
@@ -66,27 +66,6 @@ describe('HeaderSearch', function() {
       </span>);
   });
 
-  it('changes state when the close button is clicked', function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/header-search/test-header-search.js]: Looks to me like the state is still changed when the close button is clicked. Is this close button case now tested elsewhere?

**Pull request merged**